frontend/package.json

```diff
@@ -75,6 +78,7 @@
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "^27.4.2",
     "babel-loader": "^8.2.2",
+    "electron": "^22.0.0",
```

Comment:
All new dependancies should at least be dev dependencies. But I would actually recommend to create another folder outside of the frontend, eg `/electron` to create its own package json and have its own place for the main.js file. Because there is no reason to keep them in one place